.github/workflows/codespell.yml

```diff
@@ -17,6 +16,6 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
+        uses: actions/checkout@v4
```

Comment:
See this commit please
```
commit 0d7f43f110ec2a246bda906d4190767f8626446c
Author: Pedro Kaj Kjellerup Nacht <pnacht@google.com>
Date:   Wed Jul 26 13:48:07 2023 +0000

    Hash-pin workflow Actions
    
    Signed-off-by: Pedro Kaj Kjellerup Nacht <pnacht@google.com>
```

And the reason is documented here: https://github.com/libfuse/libfuse/issues/819 (should have better been part of the commit message.

Also read this please
https://blog.rafaelgss.dev/why-you-should-pin-actions-by-commit-hash

So I think we better keep the current way.

Reply:
That makes a very good point.